snippets/product-template.liquid

```diff
@@ -329,6 +339,13 @@
                         </div>
                       {%- endunless -%}
 
+                  <div class="sticky_cart ">
+                    <div class="content page-width">
+                     <h3>{{ product.title }}</h3>
+                    <button class="btn btn--full " id="add-to-cart-sticky"><span class="sticky_price">{{ product.first_available_variant.price | money }}</span> . ADD TO CART</button>
```

Comment:
use first_available_or_selected_variant so that it should display the correct price in case of any other variants are selected instead of first.